lib/SILOptimizer/Utils/SILInliner.cpp

```diff
@@ -560,25 +600,11 @@ SILValue SILInlineCloner::borrowFunctionArgument(SILValue callArg,
                                                  FullApplySite AI) {
   if (!AI.getFunction()->hasOwnership()
       || callArg.getOwnershipKind() != ValueOwnershipKind::Owned) {
-    return callArg;
+    return SILValue();
   }
 
   SILBuilderWithScope beginBuilder(AI.getInstruction(), getBuilder());
-  auto *borrow = beginBuilder.createBeginBorrow(AI.getLoc(), callArg);
```

Comment:
I moved this code to the after inline handling code. As a result, we only insert the begin_borrow here. This allowed me to handle all of the cases together.